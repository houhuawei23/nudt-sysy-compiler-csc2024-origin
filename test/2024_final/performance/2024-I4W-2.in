40
19260817
