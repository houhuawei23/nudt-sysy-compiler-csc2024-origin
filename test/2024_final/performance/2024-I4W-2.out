-71
19
0
230
205
106
-57
173
-26
200
133
62
-86
157
24
241
1
11
-231
-121
-82
-3
-77
124
91
58
-74
-24
-42
133
-18
123
36
165
-82
74
-68
126
7
201
0
